bin/json-to-sqs.js

```diff
@@ -41,23 +41,56 @@ const reader = new JSONReader({
   inputPath: jsonInputPath,
 });
 
-const sqsTransform = new SQSTransform({ queueUrl: program.url });
+const sqsTransform = new SQSTransform();
 
 let output;
 if ((program.url && program.region) || program.dryrun) {
   let sqsWriter;
+  let sqsMock;
   if (program.dryrun) {
-    sqsWriter = new SQSWriter({
-      sqs: {
-        sendMessage(message, callback) {
-          setTimeout(() => callback(), Math.random() * 5);
-        }
+    sqsMock = {
+      sendMessage(message, callback) {
```

Comment:
Nit: it might be less clutter if this was moved into a `createSqsMock()` helper function.